docs/source/migration.mdx

```diff
@@ -1504,6 +1517,10 @@ new ApolloServer({
 });
 ```
 
+### Content-Type response header
+
+In Apollo Server 3, the `Content-Type` response header is `application/json`. Apollo Server 4 this includes the encoding via the `charset` parameter: `application/json; charset=utf-8` as recommended by the [GraphQL over HTTP spec](https://github.com/graphql/graphql-over-http/blob/main/spec/GraphQLOverHTTP.md#media-types).
```

Comment:
There's something up with the grammar of the second sentence (the word "this" especially)